.github/workflows/code_style.yml

```
@@ -0,0 +1,36 @@
+name: Lint and Format
+
+on:
+  push:
+    branches: [ main ]
+
+  pull_request:
+    branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)